src/components/catalog-filter/catalog-filter.component.ts

```diff
@@ -0,0 +1,12 @@
+import {CatalogFilterController} from './catalog-filter.controller';
+
+export const catalogFilter: angular.IComponentOptions = {
+  bindings: {
+    config: '<',
+    filterOnKeyword: '<',
+    clearFilters: '<',
```

Comment:
Fixed.  When we converted a-pf to angular 1.5 components we got into the habit of not using $scope at all because it's not supported in angular 2+.